crd2pulumi-crontabs/kubernetes-go-crontabs/main.go

```diff
@@ -0,0 +1,98 @@
+package main
+
+import (
+	crontabs "kubernetes-go-crontabs/crontabs/v1"
+
+	// "github.com/pulumi/pulumi-kubernetes/sdk/v2/go/kubernetes/apiextensions"
```

Comment:
I realize this is for the "this is what you'd have to write" code block that's commented below, but I think we can delete this commented line. Right now it looks like it was intended to be deleted.

```suggestion
```